src/Codec/Xlsx/Types/StyleSheet.hs

```diff
@@ -771,6 +818,170 @@ idToStdNumberFormat i  = if i < firstUserNumFmtId then Just (NfOtherImplied i) e
 firstUserNumFmtId :: Int
 firstUserNumFmtId = 164
 
+-- | Implied cell styles
+-- Annex G.2, p. 4446
```

Comment:
I see "built-in cell styles" on that page, nothing about "implied" ones :)

Reply:
Like number formats. I also think it looks strange. If you want I can rename it to `BuiltInCellStyle`